opencart2.3.x/upload/catalog/controller/extension/payment/xendit.php

```diff
@@ -18,6 +18,43 @@ public function index() {
         return $this->load->view('extension/payment/xendit', $data);
     }
 
+    /**
+     * @param $order
+     * @return array
+     */
+    protected function extract_customer_address_object($order): array
+    {
+        $customer_address = [
+            'street_line1' => $order['payment_address_1'] ?? '',
+            'street_line2' => $order['payment_address_2'] ?? '',
+            'state' => $order['payment_zone'] ?? '',
+            'postal_code' => $order['payment_postcode'] ?? '',
+            'city' => $order['payment_city'] ?? '',
+            'country' => $order['payment_iso_code_2'] ?? ''
+        ];
```

Comment:
it's best to check each attribute, please don't make empty string as a default value since this is non acceptance criteria @andykim

Reply:
@yanoandri . I used `array_filter` to remove the empty string when returning the final array 
[here](https://github.com/xendit/xendit-opencart-module/blob/1edcd00ecc57d0cd6437e768928b05cde0edbd20/opencart2.3.x/upload/catalog/controller/extension/payment/xendit.php#L36) and [here](https://github.com/xendit/xendit-opencart-module/blob/1edcd00ecc57d0cd6437e768928b05cde0edbd20/opencart2.3.x/upload/catalog/controller/extension/payment/xendit.php#L55)